salt/pillar/sql_base.py

```diff
@@ -136,6 +136,33 @@
 The range for with_lists is 1 to number_of_fields, inclusive.
 Numbers outside this range are ignored.
 
+If you specify `as_json: True` in the mapping expression and query only for
```

Comment:
**suggestion** should mark somewhere in here that `as_json` was added in Phosphorus/3005

Reply:
right! I'll fix it now